lib/imagec/imagec_test.go

```diff
@@ -601,8 +601,8 @@ func TestFetchScenarios(t *testing.T) {
 	// valid token but image is missing we shouldn't retry
 	_, _, err = FetchImageManifest(ctx, ic.Options, 1, ic.progressOutput)
 	if err != nil {
-		// we should get a ImageNotFoundError
-		if _, imageErr := err.(urlfetcher.ImageNotFoundError); !imageErr {
+		// we should get a AccessDenied
```

Comment:
Will we ever get `ImageNotFoundErr`? The comment above notes that we have a valid token but a missing image. It surprises me that we therefore expect `AccessDenied`. I think the comment needs updating or the intent of the test needs updating.

Reply:
I have no idea where this ImageNotFoundErr came from.  I hesitate to remove something that might get displayed under a specific circumstance.  I can scan the vendored docker code to see if it's even a real message.